qtpy/tests/test_qtwebenginewidgets.py

```diff
@@ -1,10 +1,14 @@
 import pytest
+from packaging import version
 
-from qtpy import PYQT6, PYSIDE6
+from qtpy import PYQT6, PYQT_VERSION, PYSIDE6, PYSIDE_VERSION
 
 
 @pytest.mark.skipif(
-    PYSIDE6 or PYQT6,
+    not (
+        (PYQT6 and version.parse(PYQT_VERSION) >= version.parse("6.2"))
+        or (PYSIDE6 and version.parse(PYSIDE_VERSION) >= version.parse("6.2"))
+    ),
```

Comment:
I think we are missing here to add the possibility for the test to run with PyQt5 and PySide2. So this needs to be something like:

 ```suggestion
    not (
        (PYQT6 and version.parse(PYQT_VERSION) >= version.parse("6.2"))
        or (PYSIDE6 and version.parse(PYSIDE_VERSION) >= version.parse("6.2"))
        or PYQT5 or PYSIDE2
    ),
```